templates/zerver/integrations/onyx.md

```diff
@@ -0,0 +1,27 @@
+# Zulip Onyx Integration
+
+Capture discussions from Zulip channels and streams to your Onyx knowledge
+base!
```

Comment:
"Streams" is an out of date name for "channels", and doesn't belong here.

Ideally, you'd figure out how to ask the Onyx folks to fix it on their side too, if they are using it.

Reply:
I don't know what "capture" means. Do you mean "export"?

Reply:
Yes, I'd used the same word from the Onyx doc, but yes, this does sound better from the Zulip perspective, updated.